ui-speedspacechart/src/components/helpers/drawElements/speedLimitTags.ts

```diff
@@ -27,10 +27,9 @@ const alertFillBlobUrl = createSvgBlobUrl(alertFillSvg);
 
 const RECT_HEIGHT = 17;
 const Y_POSITION = 12;
-const RECTANGLE_SPACING = 1;
+const RECTANGLE_SPACING = 2;
 const TEXT_LEFT_PADDING = 4;
-const TEXT_RIGHT_PADDING = 8;
-const FIRST_TAG_LEFT_PADDING = 8;
+const TEXT_RIGHT_PADDING = 4;
```

Comment:
you can merge:
- `TEXT_LEFT_PADDING` and `TEXT_RIGHT_PADDING` to `TEXT_PADDING`
- `ICON_WIDTH` and `ICON_RIGHT` to `ICON_SIZE`
- `ICON_BACKGROUND_WIDTH` and `ICON_BACKGROUND_HEIGHT` to `ICON_BACKGROUND_SIZE`

Reply:
Yes, we talked about it but if i think it will be a problem with the logic then.
like here:
```ts
const textRectWidth = Math.min(
            actualTextWidth + TEXT_LEFT_PADDING + TEXT_RIGHT_PADDING,
            tagWidth
          );
```

Reply:
```
const textRectWidth = Math.min(
            actualTextWidth + TEXT_PADDING * 2,
            tagWidth
          );
```

Reply:
If it's more readable, no problem to change this. I thought it was clearer to understand by adding the two paddings, but indeed, apart from left and right, we don't have other options, so go for '*2'.